setup.py

```diff
@@ -66,8 +66,8 @@
     # >= 0.4.0 to be compatible with spaCy 3.
     "scispacy>=0.4.0",
     "sentence-transformers",
-    # >= 3.0.4 to include the fix for https://github.com/explosion/spaCy/issues/7352.
-    "spacy>=3.0.4",
+    # >= 3.0.6 to include the fix for https://github.com/explosion/spaCy/issues/7750.
```

Comment:
```suggestion
    # >= 3.0.6 to include the fix for https://github.com/explosion/spaCy/pull/7603.
```

Indeed:
1. 7603 is the fix while 7750 is a new feature,
2. 7750 shouldn't be needed if we use raw spaCy pipelines (decision from Tuesday in the context of #335),
3. so I would suggest to have 7603 in the comment, as it seems to stay the most relevant in the long term.